.env.example

```diff
@@ -7,6 +7,8 @@ NEXT_PUBLIC_MULTIVERSX_CHAIN = devnet
 
 # This is the masked/proxied public API endpoint
 # only current instance of the Dapp can use it if only API_ALLOWED_DAPP_HOST is set
+# Removing this line will disable the masking and make all requests go directly 
```

Comment:
I would make it optional and remove the masking comment because it can be used as a custom public endpoint different than api.elrond.com etc.

Eventually, we could add a comment that if removed, the default public Elrond API will be used, and if private `MULTIVERSX_CUSTOM_API` is present, then it should be set as proxy '/api/multiversx' or something similar.